ibmmq/json_schemas/message.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://asyncapi.com/ibmmq/bindings/message.json",
+  "title": "Message Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "patternProperties": {
+    "^x-[\\w\\d\\.\\-\\_]+$": {
+      "$ref": "https://github.com/asyncapi/asyncapi-node/blob/master/schemas/2.0.0.json#/definitions/specificationExtension"
+    }
+  },
+  "properties": {
+    "type": {
+      "type": "string",
+      "enum": ["string", "jms", "binary"]
+    },
+    "headers": {
+      "type": "string"
```

Comment:
can be present only if `type=string` or `type=jms`

Reply:
@derberg  I think you mean the opposit, in readme file it states that:
> headers MUST NOT be specified if type = string or jms

Reply:
yup, sorry 😅